ds701_book/24-NN-II-Backprop.qmd

```diff
@@ -755,11 +763,15 @@ How do we get $\partial{L}/\partial{c}$?
 
 To paraphrase from the Wikipedia page on [Chain rule](https://en.wikipedia.org/wiki/Chain_rule), if a variable $L$ depends on the variable $e$, which itself depends on the variable $c$ (that is, $e$ and $L$ are dependent variables), then $L$ depends on $c$ as well, via the intermediate variable $e$. In this case, the chain rule is expressed as
```

Comment:
I think we may want to use a proper calculus book citation, such as James Stewart, Calculus Early Transcendentals. Or we can just assume a general knowledge of the chain rule.

Reply:
have you looked at the wikipedia page? It's actually quite a good reference. Is Stewart available online?

Reply:
I don't think Stewart is (freely) available online. I think this is fine. Can always revisit at another time since we use it in previous lectures.